reaktive/src/commonMain/kotlin/com/badoo/reaktive/utils/atomic/AtomicInt.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.reaktive.utils.atomic
+
+expect class AtomicInt(initialValue: Int = 0) {
+
+    var value: Int
+
+    fun incrementAndGet(delta: Int): Int
```

Comment:
You use `1` almost everywhere, let's make `1` as default value for `incrementAndGet`.  Also originally, `incrementAndGet` does not have any parameters, because there is `getAndAdd` function (in Java implementation).

Reply:
Renamed to `addAndGet`, to be consistent with existing KN and JVM impls.